src/aspire/image/__init__.py

```diff
@@ -78,6 +79,53 @@ def _im_translate2(im, shifts):
     return im_translated
 
 
+def normalize_bg(imgs, bg_radius=1.0, do_ramp=True):
+    """
+    Normalize backgrounds and apply to a stack of images
+
+    :param imgs: A stack of images in L-by-L-by-N array
+    :param bg_radius: Radius cutoff to be considered as background (in image size)
+    :param do_ramp: When it is `True`, fit a ramping background to the data
+            and subtract. Namely perform normalization based on values from each image.
+            Otherwise, a constant background level from all images is used.
+    :return: The modified images
+    """
+    L = imgs.shape[1]
```

Comment:
why `1` and not `0`?

Reply:
Currently our implementation has shape of (L, L, N).  Probably my initial thinking is trying to make sure (N, L, L) working too.

Reply:
Fair point, but your code below these lines will break if the image was (N,L,L) anyway. Till then, better to use 0 (so the person scanning the code for refactoring will have an easier time).

Reply:
Yes. Changed in the new commit db7149e.